src/ui/components/LoadingText/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import "~amo/css/inc/vars";
+
+@keyframes placeHolderShimmer {
+  0% {
+    opacity: 0.5;
+  }
+
+  25% {
+    opacity: 0.15;
+  }
+
+  50% {
+    opacity: 0.75;
+  }
+
+  75% {
+    opacity: 0.15;
+  }
+
+  100% {
+    opacity: 0.5;
+  }
+}
+
+.LoadingText {
+  animation: placeHolderShimmer 3s infinite cubic-bezier(0.65, 0.05, 0.36, 1);
+  background: $text-color-default;
+  display: inline-block;
+  height: 1rem;
+  line-height: 1;
+  margin: 0;
+  vertical-align: middle;
+}
+
+@for $percentage from 1 through 100 {
```

Comment:
I think I'll go ahead with a client-side only solution because this loading animation is client-side only. I'll document that fact though, just in case it confuses anyone.